src/time_steppers.jl

```diff
@@ -104,6 +96,22 @@ end
 
 time_step!(model; Nt, Δt) = time_step!(model, Nt, Δt)
 
+function solve_for_pressure!(::CPU, model::Model)
+    Nx, Ny, Nz = model.grid.Nx, model.grid.Ny, model.grid.Nz
+    RHS, ϕ = model.poisson_solver.storage, model.poisson_solver.storage
```

Comment:
maybe a little late for this comment, but a better name for this variable could be `scratch` --- its not persistent memory storage, but temporary scratch memory space.